### PR TITLE
Reduce number of reference to third party libraries.

### PR DESCRIPTION
**Pull request by marcfehling**
Follow-up to https://github.com/dealii/release-papers/pull/83#discussion_r905093143.

~Please merge #83 first, I will then rebase.~ EDIT: Branch is rebased. Ready for review/merge

### CODE REVIEW

File: 9.4/paper.bib [hunk 1]
```diff
@@ -491,7 +491,8 @@ @Misc{cpp14
   howpublished = {\url{https://www.iso.org/standard/64029.html}},
   year =         2014}
 
-@Article{trilinos,
+See: https://trilinos.github.io/cite.html
```
Comment (tjhei) [inline 9.4/paper.bib:494]: The projects asks to cite the website, so I think we should keep both entries.

Comment (marcfehling) [reply 9.4/paper.bib:494]: If we just add the website for the Trilinos project, I fell it's unfair compared to the other libraries. We should be consistent.

I tried to find the best of both worlds with this PR: I added the website as a note to each publication we reference for every library (see line 505 in this patch for Trilinos). Looks like this in the paper: image

What do you all think about this approach?

Comment (tjhei) [reply 9.4/paper.bib:494]: My point is that I am okay with two citations per project if the project asks to be cited that way. Adding a project url to a citation is a bit weird on my opinion and I would rather use a footnote or a bib entry.

**Comment by marcfehling** [comment]
If I could find a note of the authors of third party libraries on how to cite their software, I adjusted our bibliography accordingly.

If I couldn't find any information, I reduced the number of references to 1: either a publication, or just a link to the website if there are no publications.

**Review by drwells — APPROVED** [review]

**Comment by drwells** [comment]
Thanks for doing all the investigating - this is close enough that I'll merge it.

**Pull request merged**